.env.example

```
@@ -52,3 +52,16 @@ VERTEX_EMBED_CONTENT_MODEL=gemini-embedding-001
 # Spreadsheet Auth
 
 OpenCRE_gspread_Auth=path/to/credentials.json
+
+# Module C — Librarian
+# Loaded by application/utils/librarian/config_loader.py. Nothing consumes these
+# yet; defaults match the OIE design doc. Constraints enforced at load time:
+# top_k_rerank <= top_k_retrieval; thresholds in [0.0, 1.0]; counts > 0.
+
+CRE_LIBRARIAN_CROSSENCODER_MODEL=cross-encoder/ms-marco-MiniLM-L-6-v2
+CRE_LIBRARIAN_TOP_K_RETRIEVAL=20
+CRE_LIBRARIAN_TOP_K_RERANK=5
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CRE_LIBRARIAN_TOP_K_RERANK key should go before the CRE_LIBRARIAN_TOP_K_RETRIEVAL key

(UnorderedKey)